pandas/core/generic.py

```diff
@@ -4455,6 +4455,9 @@ def _align_series(self, other, join='outer', axis=None, level=None,
         raise_on_error : boolean, default True
             Whether to raise on invalid data types (e.g. trying to where on
             strings)
+        apply_other : boolean, default True
+            If False, other will be stored directly rather than applied to
```

Comment:
That's fine with me; I left it in because @sinhrks wanted it for the case where "where" is called directly rather than through setitem.